src/actors/builtin/cron/cron_actor.go

```diff
@@ -1,26 +1,38 @@
 package cron
 
 import (
+	addr "github.com/filecoin-project/go-address"
 	abi "github.com/filecoin-project/specs/actors/abi"
 	builtin "github.com/filecoin-project/specs/actors/builtin"
 	vmr "github.com/filecoin-project/specs/actors/runtime"
 )
 
-func (a *CronActorCode_I) Constructor(rt vmr.Runtime) vmr.InvocOutput {
+type CronActorState struct{}
+
+type CronActor struct {
+	Entries []CronTableEntry
```

Comment:
Interesting. I think this is a bug in the original. 

Please leave a TODO to move `Entries` into the `CronActorState` struct.